dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -184,6 +184,7 @@ VoltageLevelGraphUndefined  =             VOLTAGE LEVEL %1% : graph is undefined
 VoltageLevelBBSError        =             VOLTAGE LEVEL %1% : error while looking for the closest bus bar section
 FrequencyCollapse           =             frequency collapse (%1% Hz < %2% Hz)
 FrequencyIncrease           =             high frequency (%1% Hz > %2% Hz)
+CalculatedBusNoSwitchStateChange =        calculated bus %1% cannot change state as there are no retained switches close to
```

Comment:
I would propose "calculated bus %1% cannot change state as there is no retained switch in the node"

Reply:
ok, done